ChaosMod/Effects/db/Peds/PedsSeriousKamikazes.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+	Effect by Moxi based on Killer Clowns by Last0xygen, explosives peds
+*/
+
+#include <stdafx.h>
+
+static std::list<Ped> kamikazeEnemies;
+static std::list<Object> bombObject;
+static int spawnTimer = -1;
+static Hash relationshipGroup;
+static int maxKamikazesToSpawn = 5;
```

Comment:
5 maybe a little too much for „instant killing suicide bombers“

Reply:
In my testing 5 seems ok, would you like 4?